index.js

```diff
@@ -63,6 +64,21 @@ app.post('/submit', async (req, res) => {
     res.status(200).send()
 })
 
+/**
+ * Endpoint for validation of orders. Used by the OME to validate signatures before 
+ * passing on the orders to the book. This rejects invalid orders early to avoid
+ * false liquidity on the books.
+ */
+app.post('/check', async (req, res) => {
+    if (!req.body.order || !req.body.trader || !req.body.network || !req.body.sig) {
+        console.log("missing params")
+        return res.status(500).send({ error: "Invalid params provided" })
+    }
+    let isValidSig = validateSignature(req.body.order, req.body.trader, req.body.network, req.body.sig)
+    console.log(isValidSig)
+    return res.status(200).send({ verified: isValidSig })
```

Comment:
In the event of an _invalid_ signature, the executioner will respond with a HTTP 200 OK when it should likely error.

Reply:
Likely a 400 also.

Reply:
I'd disagree here @sporejack, isn't 400 reserved for bad request? This is a valid request, the result is just the signature is invalid but the server acted as expected

Reply:
@raymogg This is a fair enough call in my opinion (and I did contemplate this before making this comment), but it stills feels awkward having an invalid signature return as a success. The alternative is to just boot it up the protocol abstraction layer (which is the current state of things on this line).